generation/core/global-functions.js

```diff
@@ -127,6 +127,10 @@ function sanitize_uiAccessibilityTraits(value) {
 } // END sanitize_uiAccessibilityTraits
 
 function sanitize_matcher(matcher) {
+  if (!matcher._call) {
```

Comment:
Looks something new to me, can you explain a bit? Is that a fix to the failing unit tests back then?